cli/tests/integration_tests.rs

```diff
@@ -1318,6 +1343,32 @@ fn cafile_bundle_remote_exports() {
   drop(g)
 }
 
+#[test]
```

Comment:
@ry Should I modify this test to be able to run them in parallel? Adding the permission, as a suffix, to the function name.

Reply:
prefix is preferable

Reply:
actually the way you have it is fine. i think it's easier to read this way.